templates/federated_reporting/distributed_cleanup.py

```diff
@@ -120,11 +128,16 @@ def interactive_setup_feeder(hub, email, fr_distributed_cleanup_password):
         sys.exit(1)
 
 
-def interactive_setup():
-    fr_distributed_cleanup_password = "".join(random.choices(string.printable, k=20))
-    admin_pass = getpass(
-        prompt="Enter admin password for superhub {}: ".format(socket.getfqdn())
-    )
+def interactive_setup(force_interactive=False):
+    fr_distributed_cleanup_password = "".join(random.choices(string.digits + string.punctuation + string.ascii_letters, k=20))
```

Comment:
I wonder if other symbols in `string.punctuation` can cause similar issues in API usage? If it goes through HTTP request, they're supposed to be %-encoded-decoded and that's where the space issue might be coming from. Maybe it's safer to use `string.digits + string.ascii_letters`? I believe we can increase `k=20` to something bigger for extra security :)

Reply:
ok! Nah, no need to investigate too much, I was just wondering if any of the "funny characters" in `string.punctuation` can cause us an issue. But finding why exactly space causes the breakage equals to fixing it :)

I'm just thinking: maybe password gets pasted into some shell command, like
```
sh -c "cleanup.sh $USER $PASSWORD"
```
then not only space, but a pipe character will also cause breakage!

Reply:
I just removed punctuation, should be OK without it.